.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,35 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: 'bug'
+assignees: ''
```

Comment:
So also to note: we could add a default assignee here. I opted not to because I get notifications for all activity (so critical bugs shouldn't go under the radar).

We can set this up for Peter at handover too.